arch/x86/deflate_quick.c

```diff
@@ -224,7 +224,7 @@ ZLIB_INTERNAL block_state deflate_quick(deflate_state *s, int flush) {
         }
 
         if (s->lookahead < MIN_LOOKAHEAD) {
-            fill_window_sse(s);
+            functable.fill_window(s);
```

Comment:
These changes do not seem to belong here.

Reply:
They belong there because without them it is not possible to do `-WITH_SSE2=OFF`

Reply:
Ok I have removed them.